app.js

```diff
@@ -12,14 +12,30 @@ app.set('views', path.join(__dirname, 'views'));
 
 app.use(express.static(path.join(__dirname, 'public')));
 
-// Register the location for handlebars partials here:
-
-// ...
-
-// Add the route handlers here:
+hbs.registerPartials(path.join(__dirname, '/views/partials'));
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then(beersFromApi => {
+    console.log('Beers from the database: ', beersFromApi);
+    res.render('beers', {beersFromApi});
+  })
+  .catch(error => console.log(error));  
+})
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+  .getRandom()
+  .then(responseFromAPI => {
+    console.log('Random beer ', responseFromAPI);
+    res.render('random-beer', {responseFromAPI});
+  })
+  .catch(error => console.log(error));
+})
+
```

Comment:
It's good to remove console logs after you're done with them / when handing in a project.